docs/Using-Fleet/REST-API.md

```diff
@@ -1682,6 +1683,29 @@ None.
 - [Get host OS versions](#get-host-os-versions)
 - [Get hosts report in CSV](#get-hosts-report-in-csv)
 
+### On the different timestamps in the host data structure
+
+Hosts have a set of timestamps usually named with an "_at" suffix, such as created_at, enrolled_at, etc. Before we go 
+through each of them and what they mean, we need to understand a bit more about how the host data structure is 
+represented in the database.
+
+The table `hosts` is the main one. It holds the core data for a host. A host doesn't exist if there is no row for it in 
+this table. This table also holds most of the timestamps, but it doesn't hold all of the host data. This is an important
+detail as we'll see below.
+
+There's adjacent tables to this one that usually follow the name convention `host_<extra data descriptor>`. Examples of 
+this are: `host_additional` that holds additional query results, `host_software` that links a host with many rows from 
+the `software` table.
+
+- created_at: the time the row in the database was created, which usually corresponds to the first enrollment of the host.
+- updated_at: the last time the row in the database for the `hosts` table was updated.
+- detail_updated_at: the last time we updated the host data (any of it, regardless of the location) based on the results from the detail queries.
```

Comment:
"regardless of the location", do you mean "regardless of the location of the host data in the database"?